package.json

```diff
@@ -8,7 +8,8 @@
     "build": "astro check && astro build && partykit deploy",
```

Comment:
This doesn’t have to be for this PR. It can be a small follow up PR, but it’d be good to have a `prebuild` script that runs `vitests run`, so if there is a test failure the build will fail.